src/contexts/cartContext.js

```diff
@@ -0,0 +1,22 @@
+import { createContext, useContext, useReducer } from "react";
+import cartReducer from "../reducers/cartReducer";
+
+const CartContext = createContext();
+
+const CartProvider = ({ children }) => {
+  const [state, dispatch] = useReducer(cartReducer, {
+    cart: [],
+    wishList: [],
+    quantity: 0,
+    totalPrice: 0,
+  });
```

Comment:
Suggestion :  You can create an initial cartContextdata object and use it to initialize the context and also as a second argument for the reducer